app/models/game.rb

```diff
@@ -6,4 +6,5 @@ class Game < ApplicationRecord
   belongs_to :black_player, class_name: 'User'
 
   enum game_status: { in_progress: 0, checkmate: 1, stalemate: 2 }
+  scope :available, -> { where(available: true) }
```

Comment:
I got you :-)
Created a new migration to delete the available attribute and changes the scope to:
scope :available, -> { where('black_player_id = ? OR white_player_id = ?', nil, nil) }